.github/actions/argus-builder/build-prep/action.yml

```diff
@@ -29,37 +29,19 @@ outputs:
   should_build:
     description: Whether the job should run
     value: ${{ steps.final_check.outputs.should_build }}
+  base:
+    description: The base branch or commit to compare against
+    value: ${{ steps.force_push.outputs.base }}
```

Comment:
maybe `base_ref` might be a more descriptive name. `base` is an overloaded term we use a lot